client/components/seo/facebook-preview/style.scss

```diff
@@ -33,6 +33,7 @@
 	color: #1d2129;
 	font-family: Georgia, serif;
 	font-size: 18px;
+	/* stylelint-disable-next-line scales/font-weight */
```

Comment:
Why cannot use our enforced values here?

Reply:
This component mimics facebook styles in the post preview, though it probably hasn't been updated in quite a while

Reply:
Pinging @dmsnell since your notes at the top of this file suggest we shouldn't update these styles for preview parity; do you know if that's still the case here?

Reply:
You should be fine here @sixhours, :shipit:

Reply:
thanks for the ping and thanks for checking the comment!

the intention of the comment is to say that the styles in this file are not meant to harmonize with Calypso. They are specifically here in order to mimic to the best of our ability what the previews on Facebook and Twitter _actually_ look like vs. some form of "Calypsoified Facebook."

If the font-weight here helps make the social previews match more closely to shared posts on Facebook then go for it. If the existing font-weight more closely matches what shared posts on Facebook look like then that's what determines if we should keep it.